swagger-spec/registrations/wikis_list.yaml

```diff
@@ -0,0 +1,92 @@
+# /registrations/{registration_id}/wikis/
+get:
+  summary: List all wikis
+  description: >-
+    A paginated list of the registration's wiki pages
+
+    ####Returns
+
+    A list of all registration's current wiki page versions ordered by their date_modified.
+    Each resource contains the full representation of the wiki, meaning additional requests
+    to an individual wiki's detail view are not necessary.
+
+
+    Note that if an anonymous view_only key is being used, the user relationship will not be exposed.
+
+
+    If the request is unsuccessful, a JSON object with an `errors` key containing
+    information about the failure will be returned. Refer to the [list of error codes](#Introduction_error_codes)
+    to understand why this request may have failed.
+
+    #### Filtering
+
+    Wiki pages can be filtered based on their `name` and `date_modified` fields.
+
+    + `filter[name]=<Str>` -- filter wiki pages by name
+
+    + `filter[date_modified][comparison_operator]=YYYY-MM-DDTH:M:S` -- filter wiki pages based on date modified.
+
+
+    Possible comparison operators include 'gt' (greater than), 'gte'(greater than or equal to), 'lt' (less than) and 'lte' (less than or equal to). The date must be in the format YYYY-MM-DD and the time is optional.
+
+  parameters:
+    - in: path
+      type: string
+      name: registration_id
+      required: true
+      description: 'The unique identifier of the registration.'
+  tags:
+    - Registrations
+  operationId: registrations_wikis_list
+  x-response-schema: Wiki
+  responses:
+    '200':
+      description: OK
+      schema:
+        type: array
+        items:
+          $ref: '../wikis/definition.yaml'
+      examples:
+        application/json:
+          data:
+            - relationships:
+                node:
+                  links:
+                    related:
+                      href: 'https://api.osf.io/v2/nodes/tvyxz/'
```

Comment:
Can you grab an example that's a registration wikis response? Just so these links are `/v2/registrations/.../` instead of `/v2/nodes/.../`.